Frontend/student-hub/src/pages/CoursePage.js

```diff
@@ -1,8 +1,8 @@
 import React, { useState, useEffect } from 'react';
-import { Grid, Typography, Container, Button, TextField, Card, CardContent, CardActions, IconButton, Paper, InputBase, Dialog, DialogTitle, DialogContent, DialogActions } from '@mui/material';
+import { Grid, Typography, Container, Button, TextField, Card, CardContent, CardActions, IconButton, Paper, InputBase, Dialog, DialogTitle, DialogContent, DialogActions, MenuItem, Select } from '@mui/material';
 import { useNavigate } from 'react-router-dom';
 import SearchIcon from '@mui/icons-material/Search';
-import ThumbUpIcon from '@mui/icons-material/ThumbUp';
+//import ThumbUpIcon from '@mui/icons-material/ThumbUp';
```

Comment:
delete comment (think I saw a commit which did that but the merge conflicts probably re-introduced this)